README.md

```diff
@@ -19,19 +19,21 @@ English | [简体中文](./i18n/README.zh-cn.md) | [繁體中文](./i18n/README.
 
 # QuestDB
 
-QuestDB is a high-performance, open-source SQL database for applications in
-financial services, IoT, machine learning, DevOps and observability. It includes
-endpoints for PostgreSQL wire protocol, high-throughput schema-agnostic
-ingestion using InfluxDB Line Protocol, and a REST API for queries, bulk
-imports, and exports.
-
-QuestDB implements ANSI SQL with native extensions for time-oriented language
-features. These extensions make it simple to correlate data from multiple
-sources using relational and time series joins. QuestDB achieves high
-performance from a column-oriented storage model, massively-parallelized vector
-execution, SIMD instructions, and various low-latency techniques. The entire
-codebase was built from the ground up in Java and C++, with no dependencies, and
-is 100% free from garbage collection.
+QuestDB is an open-source time-series database for high throughput ingestion and
+fast SQL queries with operational simplicity. It supports schema-agnostic
+ingestion using the InfluxDB line protocol, PostgreSQL wire protocol, and a REST
+API for bulk imports and exports.
+
+QuestDB is well suited for financial market data, application metrics, sensor
+data, realtime analytics, dashboards, and infrastructure monitoring.
+
+QuestDB uses ANSI SQL and benefits from time-series SQL semantics. These SQL
+semantics make it simple to correlate data from multiple sources using
+relational and time-series joins. QuestDB achieves high performance from a
+column-oriented storage model, parallelized vector execution, SIMD instructions,
+and various low-latency techniques. The entire codebase is built from the ground
+up in Java and C++, with no dependencies, and is 100% free from garbage
+collection.
```

Comment:
trying to simplify these sentences, reduce repetition.

```suggestion
QuestDB uses ANSI SQL and benefits from time-series SQL semantics. They
make it easy to correlate data from multiple sources using relational
and time-series joins.

High performance is achieved from a column-oriented storage model,
parallelized vector execution, SIMD instructions and various low-latency
techniques. The entire codebase is built from scratch in Java and C++,
with no dependencies, and is 100% free from garbage collection.
```

Reply:
Thanks! for the second paragraph tho I'd still avoid passive voice (which is generally not recommended in technical writing.)

Reply:
Id like to avoid every paragraph starting with `QuestDB`, looks very monotonic, that's why choice different phrase

Reply:
@argshook  brainstorming, how about

> We optimize the performance using column-oriented storage model, parallelized vector execution, SIMD instructions, and various low-latency techniques.

or 

> We achieve high performance using column-oriented storage model, parallelized vector execution, SIMD instructions, and various low-latency techniques.

Reply:
Maybe the second one?

I updated my code suggestion with that one, if you'd choose to use it